tests/index.ts

```diff
@@ -0,0 +1,33 @@
+import { Application } from '../application.ts'
+import { assertEquals } from "https://deno.land/std/testing/asserts.ts";
+
+const port = 8080
+const app = new Application({ port: 8080 })
+
+app.start()
+
+async function request(suffix: string = '') {
```

Comment:
this function needs to be flexible in methods, you can pass a parameter for it like "method";
```suggestion
async function request(suffix: string = '', method: "GET" | "POST" ) {
```